internal/grpc/services/gateway/storageprovider.go

```diff
@@ -1437,13 +1447,7 @@ func (s *svc) findProviders(ctx context.Context, ref *provider.Reference) ([]*re
 	switch {
 	case ref == nil:
 		return nil, errtypes.BadRequest("missing reference")
-	case ref.ResourceId != nil: // can we use the provider cache?
-		// only the StorageId is used to look up the provider. the opaqueid can only be a share and as such part of a storage
-		if value, exists := s.providerCache.Get(ref.ResourceId.StorageId); exists == nil {
-			if providers, ok := value.([]*registry.ProviderInfo); ok {
-				return providers, nil
-			}
-		}
+	case ref.ResourceId != nil:
```

Comment:
Is this case intended to do nothing? If so could you add a comment?

Reply:
Exactly. No Action needed. I added a comment